ethereum_history_api/oracles/src/noir/oracles/transactionOracle/encode.ts

```diff
@@ -10,7 +10,8 @@ import { removeHexPrefix } from '../../../util/hex.js';
 const MAX_TX_KEY_LEN = 3;
 export const MAX_TX_KEY_NIBBLE_LEN = 6;
 export const MAX_TX_TREE_DEPTH = MAX_TX_KEY_NIBBLE_LEN + 1;
-export const MAX_TX_RLP_LEN = MAX_TRIE_NODE_LEN;
+export const MAX_TX_ENCODED_LEN = 525;
+export const MAX_TX_RLP_LEN = MAX_TX_ENCODED_LEN - 1;
```

Comment:
Why is MAX_TX_RLP_LEN == 524 and not 523 as it was before?

Reply:
To be consistent with receipts:
```
export const MAX_RECEIPT_ENCODED_LEN = 525;
export const MAX_RECEIPT_RLP_LEN = MAX_RECEIPT_ENCODED_LEN - 1;
```

But I'm not sure. Is there an argument for 524 instead of 525?